wagtail_transfer/field_adapters.py

```diff
@@ -84,17 +86,48 @@ def get_object_references(self, instance):
         return get_object_references(stream_block, stream)
 
 
+class ManyToManyFieldAdapter(FieldAdapter):
+    def __init__(self, field):
+        super().__init__(field)
+        self.related_base_model = get_base_model(self.field.related_model)
+
+    def _get_pks(self, instance):
+        return [model.pk for model in self.field.value_from_object(instance)]
+
+    def get_object_references(self, instance):
+        refs = set()
+        for pk in self._get_pks(instance):
+            refs.add((self.related_base_model, pk))
+        return refs
+
+    def serialize(self, instance):
+        pks = list(self._get_pks(instance))
+        return pks
+
+
+class ParentalManyToManyFieldAdapter(ManyToManyFieldAdapter):
+    def __init__(self, field):
+        super().__init__(field)
+        self.related_base_model = get_base_model(self.field.related_model)
+
+    def _get_pks(self, instance):
+        return self.field.value_from_object(instance).values_list('pk', flat=True)
```

Comment:
Ah yes, didn't spot that because I did parental first, and regular `ManyToMany` wouldn't work with `values_list`. Just tried removing and it works just fine, so yep, let's drop it!